core/components/atoms/collapsible/__stories__/CustomTrigger.story.jsx

```diff
@@ -1,23 +1,20 @@
 import * as React from 'react';
```

Comment:
please check if it is required to add more data/context in this story as it is mention in the story to add some more content inside this.
![Uploading Screenshot 2025-05-07 at 1.06.38 PM.png…]()